base/mpfr.jl

```diff
@@ -37,7 +37,7 @@ function __init__()
     end
 end
 
-const ROUNDING_MODE = Cint[0]
+const ROUNDING_MODE = Ref{Cint}(0)
```

Comment:
OK, it's a valid concern though. I tried to look for such packages (Google, codesearch) but couldn't find any.